lib/types/operation.js

```diff
@@ -694,7 +694,7 @@ Operation.prototype.execute = function (arg1, arg2, arg3, arg4, parent) {
     }
   };
 
-  this.clientAuthorizations.apply(obj, this.operation.security);
+  this.clientAuthorizations.apply(obj, this.security);
```

Comment:
This is going to need some testing.  If you can add anything around this to ensure we're not introducing a regression on operation-based security, we can move forward.